### DISCUSSION TITLE
dynamic publicPath doesn't work correctly.

### DISCUSSION THREAD

**Discussion by hassan-jahan** [nuxt/nuxt#10657] [Questions (Nuxt 2)]
Hi-

Nuxt 2.15 has come with dynamic publicPath but I couldn't make it work. 

I have something like this on my nuxt.config:
```
build: {
     publicPath: (process.env.BASE_URL || 'http://localhost:3000') + '/_nuxt/'
}
```
While it works after building it doesn't update BASE_URL during runtime which breaks the page. It seems it's because process.env is not available during runtime. Any idea?

**Comment by Nikolinjho** [comment]
@hassan-jahan did you find any solution?
